src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/email/AppellantAppealOutcomePersonalisationEmail.java

```diff
@@ -0,0 +1,86 @@
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.service.RecipientsFinder;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.CustomerServicesProvider;
+
+
+@Service
+public class AppellantAppealOutcomePersonalisationEmail implements EmailNotificationPersonalisation {
+
+    private final String appealOutcomeAllowedAppellantTemplateId;
+    private final String appealOutcomeDismissedAppellantTemplateId;
+    private final String iaAipFrontendUrl;
+    private final RecipientsFinder recipientsFinder;
+    private final CustomerServicesProvider customerServicesProvider;
+
+    public AppellantAppealOutcomePersonalisationEmail(
+        @NotNull(message = "appealOutcomeAllowedAppellantTemplateId cannot be null") @Value("${govnotify.template.appealOutcomeAllowed.appellant.email}") String appealOutcomeAllowedAppellantTemplateId,
+        @NotNull(message = "appealOutcomeDismissedAppellantTemplateId cannot be null") @Value("${govnotify.template.appealOutcomeDismissed.appellant.email}") String appealOutcomeDismissedAppellantTemplateId,
+        @Value("${iaAipFrontendUrl}") String iaAipFrontendUrl,
+        RecipientsFinder recipientsFinder,
+        CustomerServicesProvider customerServicesProvider
+    ) {
+        this.appealOutcomeAllowedAppellantTemplateId = appealOutcomeAllowedAppellantTemplateId;
+        this.appealOutcomeDismissedAppellantTemplateId = appealOutcomeDismissedAppellantTemplateId;
+        this.iaAipFrontendUrl = iaAipFrontendUrl;
+        this.recipientsFinder = recipientsFinder;
+        this.customerServicesProvider = customerServicesProvider;
+    }
+
+    @Override
+    public String getTemplateId(AsylumCase asylumCase) {
+
+        final AppealDecision appealOutcomeDecision = asylumCase
+            .read(AsylumCaseDefinition.IS_DECISION_ALLOWED, AppealDecision.class)
+            .orElseThrow(() -> new IllegalStateException("appealOutcomeDecision is not present"));
+
+        return appealOutcomeDecision.getValue().equals(AppealDecision.ALLOWED.getValue())
+            ? appealOutcomeAllowedAppellantTemplateId : appealOutcomeDismissedAppellantTemplateId;
+    }
+
+    @Override
+    public Set<String> getRecipientsList(final AsylumCase asylumCase) {
+        return recipientsFinder.findAll(asylumCase, NotificationType.EMAIL);
+    }
+
+    @Override
+    public String getReferenceId(Long caseId) {
+        return caseId + "_APPEAL_OUTCOME_AIP_APPELLANT_EMAIL";
+    }
+
+    @Override
+    public Map<String, String> getPersonalisation(AsylumCase asylumCase) {
+        requireNonNull(asylumCase, "asylumCase must not be null");
+
+        YesOrNo appellantInUK = asylumCase.read(AsylumCaseDefinition.APPELLANT_IN_UK, YesOrNo.class).orElse(YesOrNo.NO);
+
+        String period = appellantInUK == YesOrNo.YES ? "14" : "28";
```

Comment:
@burale this is incorrect when looking at your GovNotify template id `1d3266fe-ec61-4bd2-a7de-50092ec557ff`

If you have as `14` then your GovNotify template needs to change. Right now, the email going out is going to read:

`If you think the decision to dismiss your appeal is wrong, you have until 14 after judge uploads decisions and reasons document)) to ask for permission to appeal to the Upper Tribunal.`